### DISCUSSION TITLE
Readability Color Coding

### DISCUSSION THREAD

**Discussion by lochana** [Zettlr/Zettlr#4196] [Q&A]
I am extremely confused about the color coding in readability mode. I have various shades of blue, yellow and brown. Not much green and red. Can somebody explain what these colors suppose to mean. Why can't Zettlr display a readability score with limited number of colors?

**Comment by nathanlesage** [comment]
> Why can't Zettlr display a readability score with limited number of colors?

Originally, it did. I used HSV hues to switch colors smoothly from red to green, but somehow this got botched without me noticing. I noticed this bug a few weeks ago, and then re-implemented more useful colors (red -> yellow -> purple -> green), but that's only available in the betas and/or nightlys.

**Reply by lochana** [reply]
Thanks for getting back to me. Tried latest beta and nightly (on windows) as well. I still see orange and blue colors. Colors also disappear when I scroll fast. Colors seem to be incorrect around markdown formattings like bold and italic.
